pkg/components/state/pluggable_test.go

```diff
@@ -258,7 +258,7 @@ func TestComponentCalls(t *testing.T) {
 		stStore, cleanup, err := getStateStore(svc)
 		require.NoError(t, err)
 		defer cleanup()
-		err = stStore.Delete(&state.DeleteRequest{
+		err = stStore.Delete(context.TODO(), &state.DeleteRequest{
```

Comment:
Please change all `context.TODO`'s in test (all test files) to `context.Background`

Reply:
Sorry. I've just done an entire replacement using `sed` command.